presentation/common/src/presentation-common/rules/hierarchy/InstanceNodesOfSpecificClassesSpecification.ts

```diff
@@ -24,6 +24,11 @@ export interface InstanceNodesOfSpecificClassesSpecification extends ChildNodeSp
    */
   classes: MultiSchemaClassesSpecification | MultiSchemaClassesSpecification[];
 
+  /**
+   * Specifications of ECClasses whose instances should be excluded.
+   */
+  excludedClasses?: MultiSchemaClassesSpecification | MultiSchemaClassesSpecification[];
+
   /**
    * Should all [[classes]] be handled polymorphically.
    */
```

Comment:
- I'd add this: <code>If set, acts as default value for `classes.arePolymorphic` and `excludedClasses.arePolymorphic`.</code>
- Should be marked `@deprecated`